packages/core/src/render3/instructions/listener.ts

```diff
@@ -217,6 +233,8 @@ export function listenerInternal(
       processOutputs = false;
     } else {
       listenerFn = wrapListener(tNode, lView, context, listenerFn);
+      const appId = lView[INJECTOR].get(APP_ID);
+      const stashEventListener = stashEventListeners.get(appId)!;
       stashEventListener(native, eventName, listenerFn);
```

Comment:
The non null assertion deviates from a prior safety net where an empty closure would be used, so the following seems safer and less of a change:

```suggestion
      const stashEventListener = stashEventListeners.get(appId);
      stashEventListener?.(native, eventName, listenerFn);
```

Reply:
Updated to include optional chaining.